checkout/wasm/payment-customization/shopify.function.extension.toml.liquid

```diff
@@ -0,0 +1,10 @@
+name = "{{name}}"
+type = "{{extensionType}}"
+api_version = "2022-07"
+
+[build]
+command = "echo 'build the wasm'"
```

Comment:
🤔 Is this supposed to be filled out?

Reply:
Putting on the hat of a 3rd party developer, seeing this would be very confusing. Is it being replaced when we scaffold?

Reply:
It's not being replaced. What do you think we should do here? cc @nickwesselman

Reply:
Hm- no particular need to deal with that in this PR 🤔 I'll resolve given that there's no clear solution here I can see

Reply:
This is for the wasm-first scaffold right? It's intentionally a no-op, as it's up to the developer in that case to configure the build command and output path based on their chosen language.